src/main/scala/akka/persistence/cassandra/journal/CassandraJournal.scala

```diff
@@ -32,6 +38,15 @@ class CassandraJournal(cfg: Config) extends AsyncWriteJournal with CassandraReco
 
   val config = new CassandraJournalConfig(cfg)
   val serialization = SerializationExtension(context.system)
+  val pubsubInterval = FiniteDuration(config.pubsubMinimumInterval.toMillis, TimeUnit.MILLISECONDS)
+
+  // PubSub will be ignored when clustering is unavailable
+  val pubsub = Try(DistributedPubSub(context.system).mediator).toOption.filter { _ =>
```

Comment:
`DistributedPubSub(context.system).mediator` will only throw if ClusterActorRefProvider is not configured, but we might also want to check `DistributedPubSub(context.system).isTerminated`